src/layout/header/Header.tsx

```diff
@@ -157,7 +158,7 @@ export const Header = () => {
           {!isMobile && (
             <>
               {organization?.acronym &&
-                (user?.isEditor ? (
+                (hasCustomer ? (
```

Comment:
Typography under her kan vel bare tas vekk nå?
```
<Typography
                    variant="h1"
                    component="span"
                    sx={{ whiteSpace: 'nowrap', color: 'inherit', alignSelf: 'center' }}>
                    {organization.acronym}
                  </Typography>
```

Reply:
Yeah, var litt usikker på om vi skulle ha en form for fallback eller ikke. Fjerner 👍

Reply:
`organization?.acronym` gjør faktisk stort sett samme sjekken da 🤔 
kan sikkert slå sammen dem til `organization?.acronym && hasCustomer`, selv om de i praksis vil være det samme